scripts/precompile-headers.sh

```diff
@@ -40,16 +40,16 @@ VERSION=$(${HILTI_CONFIG} --version | cut -d ' ' -f1)
 # if set; else a patch under the user's home directory is used.
 CACHE=${SPICY_CACHE:-$HOME/.cache/spicy/${VERSION}}
 
-echo "Clearing cache directory $CACHE"
+# echo "Clearing cache directory $CACHE"
```

Comment:
Let's just get rid of these lines instead of commenting them out.

Here and below.